mesa/space.py

```diff
@@ -80,38 +85,43 @@ def __init__(self, width, height, torus):
         self.width = width
         self.torus = torus
 
-        self.grid = []
+        self.grid = []  # type: List[List[Any]]
 
-        for x in range(self.width):
+        for _ in range(self.width):
             col = []
-            for y in range(self.height):
+            for __ in range(self.height):
                 col.append(self.default_val())
             self.grid.append(col)
 
         # Add all cells to the empties list.
         self.empties = list(itertools.product(
-                            *(range(self.width), range(self.height))))
+            *(range(self.width), range(self.height))))
 
     @staticmethod
-    def default_val():
+    def default_val() -> None:
         """ Default value for new cell elements. """
         return None
 
-    def __getitem__(self, index):
+    def __getitem__(self, index: int) -> List[Any]:
         return self.grid[index]
 
-    def __iter__(self):
-        # create an iterator that chains the
-        #  rows of grid together as if one list:
+    def __iter__(self) -> Any:
+        # TODO precisify the Mypy type annotation
```

Comment:
Precisify? In terms of the grid values? I'm not sure what this means.

Reply:
Yes, it should be more specific that a generic `Any`, it should be `Iterator[Set[Any]]` maybe.

Reply:
That looks like a good annotation, though for the base `Grid` class I wonder that since `default_val` returns `None`, though others return `set` (and potentially something different than either of those), that maybe `Iterator[Any]` might suffice.

Reply:
Ok, will put that in.